modules/service/data-service.js

```diff
@@ -109,14 +109,14 @@ class DataService {
 
     async resolve(id, localQuery = false, metadataOnly = false) {
         try {
-            let {nquads, isAsset} = await this.implementation.resolve(id);
+            let nquads = await this.implementation.resolve(id);
             if (!localQuery && nquads && nquads.find((x) => x.includes(`<${constants.DID_PREFIX}:${id}> <http://schema.org/hasVisibility> "private" .`))) {
```

Comment:
Lets add a todo for things like this please, it's quite unreadable. I suggest moving to a method